addons/hr_work_entry/models/hr_work_entry.py

```diff
@@ -187,7 +188,12 @@ def write(self, vals):
         with self._error_checking(skip=skip_check, employee_ids=employee_ids):
             return super(HrWorkEntry, self).write(vals)
 
+    def _error_raising(self):
```

Comment:
The method name is too vague, on the other hand, I don't see why we should isolate this into a specific method for a simple check. 

On the other hand, the work entry could be validated for another reason than being used in a validated payslip (+ payroll is not even installed in this context).